main/src/main/java/cgeo/geocaching/maps/MapSettingsUtils.java

```diff
@@ -151,7 +151,7 @@ public static void showSettingsPopup(final Activity activity, @Nullable final In
             dialogView.mapSettingsProximitynotificationMastertoggle.setChecked(Settings.isProximityNotificationMasterToggleOn());
         }
 
-        final Dialog dialog = Dialogs.bottomSheetDialogWithActionbar(activity, dialogView.getRoot(), R.string.quick_settings);
```

Comment:
We could fix `Dialogs.bottomSheetDialogWithActionbar` to use `context.getString()` to fix potential other places where this error occurs.

Reply:
I will check the calls and change it accordingly

Reply:
Alternatively here is how you could get a localized context from a given context (say, the application context inside LocalizatioUtils :-))

```
// Create a new Configuration object with the specified locale
Configuration configuration = new Configuration(resources.getConfiguration());
configuration.setLocale(locale);

// Create a new Context with the updated configuration
Context localizedContext = context.createConfigurationContext(configuration);
```

Reply:
To be honest, I didn't want to change to much before it's not verified, that this solution solves the problem. 

But then I would probably try the variant with the new localized context....